.github/workflows/lint.yml

```diff
@@ -41,3 +41,14 @@ jobs:
             echo "ERROR: 'go mod tidy' modified the source tree."
             exit 1
           fi
+
+      - name: make manifests
```

Comment:
does it make sense to combine this in some way with the previous task (e.g. is the working directory reset? would the state of the repo be dirty when this starts? if go mod tidy fails, does this task even run?)

Reply:
IMO it's worth keeping separate so that we can tell which step modified the repo.

> is the working directory reset?

Nope

> would the state of the repo be dirty when this starts? if go mod tidy fails, does this task even run?

No to both